src/schemata/definitions/field_schema/timer.yml

```
@@ -1,31 +1,45 @@
 ---
 type: object
 description: |
-  Defines a field of type `timer`.
+  Defines a field of type `timer`. Timers are used to measure time or execute time-controlled actions within a Workflow Run.
+
+  Timers have two possible modes of operation: countdown and stopwatch.
+  
+  In **countdown** mode, the timer will run for a specified duration, and fire a `on_timer_complete` Trigger upon completion.
+  In **stopwatch** mode, the timer measures time until it is stopped, either manually or through a Behavior.
+  
+  Timer fields can be assigned to individual substeps by passing the field identifier to the `timer` property.
 required: [type]
 properties:
   type:
     type: string
     enum: [timer]
   defaultDuration:
-    description: A fixed duration.
+    description: |
+      The default value set on the timer upon starting the workflow. If this is defined, the timer will run in countdown mode, unless the value is reset during workflow execution.
+
+      The initial value of the timer can also be set using the `defaultValue` property. If both are defined, `defaultValue` will overwrite the value set in `defaultDuration`.
```

```suggestion
    description: |
      The default value of the timer. If set to a non-zero value, the timer will run in countdown mode. If no duration is defined or it is set to _"0:00"_, the timer will run in stopwatch mode.

      Note that also defining `defaultValue` will overwrite this setting, but does not support natural language parsing of the value.
    examples:
      - 90
      - 1 mins 30 sec
      - 1:30
```
let's move these two below `$ref: '#/definitions/duration'`

---

This I think is correct, are you sure about `defaultValue` not parsing the natural language duration?

---

Tested it, it does parse the value in the same way